src/Arcus.Messaging.Pumps.ServiceBus/KeyRotation/IAzureServiceBusManagementAuthentication.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Threading.Tasks;
+using Microsoft.Azure.Management.ServiceBus;
+
+namespace Arcus.Messaging.Pumps.ServiceBus.KeyRotation
```

Comment:
This is bigger than just rotation as well:
```suggestion
namespace Arcus.Messaging.Pumps.ServiceBus
```